/*
package test;

import sun.misc.Cleaner;

import java.io.IOException;

*/
/**
 * Created by SiyuanZeng's on 12/6/2016.
 *//*


        import java.io.IOException;


public class Test{


        public static void main(String[] args) {

            Test t = new Test();

            try{t.check();

            }catch(Exception e){


                System.out.println("catch block 1");

            }

        }

        void check()throws Exception{

            try{

                throw new IOException();

            }catch(IOException e){

                System.out.println("catch block 2");

                throw new Exception();

            }


 catch(Exception e){

                System.out.println("catch block 3");

            }finally{

                System.out.println("finally returning");
            }


        }


    }

*/
